src/monali_Assignment_7/shop.java

```diff
@@ -0,0 +1,99 @@
+package monali_Assignment_7;
+
+public class shop {
+	
+	String maggiepacket;
+	int maggiepacketQ;
+	
+	String dosa;
+	int dosaQ;
+	
+	String pouches;
+	int pouchesQ;
+	
+	String panipuri;
+	int panipuriQ;
+	
+	String masala;
+	int masalaQ;
+	
+	void initialQuantity() {
+		maggiepacketQ = 50;
+		dosaQ = 75;
+		pouchesQ = 64;
+		panipuriQ = 85;
+		masalaQ = 35;
+	}
+	
+	void reqQuantity(int maggie1 , int dosa2 , int pouches3 , int panipuri4 ,int masala5){
+		maggiepacketQ = maggiepacketQ - maggie1;
```

Comment:
Here, you should check the requested number is more than the available or not, right?